src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java

```diff
@@ -263,44 +259,49 @@ public String getCorrespondenceAddress() {
                 )
                 .filter(value -> value != null && !value.isEmpty())
                 .collect(joining("\n"));
+        } else {
+            return Stream.of(
+                    address.getAddressLine1(),
+                    address.getAddressLine2(),
+                    address.getAddressLine3(),
+                    address.getPostTown(),
+                    address.getCounty(),
+                    address.getCountry(),
+                    address.getPostCode()
+                )
+                .filter(value -> value != null && !value.isEmpty())
+                .collect(joining("\n"));
         }
-
-        return null;
     }
 
     @JsonIgnore
-    public String getCorrespondenceAddressWithoutConfidentialCheck() {
+    public String getCorrespondenceAddress() {
         if (isRepresented()) {
             return Stream.of(
                     Optional.ofNullable(solicitor.getOrganisationPolicy())
                         .map(OrganisationPolicy::getOrganisation).map(Organisation::getOrganisationName).orElse(null),
                     solicitor.getAddress()
                 ).filter(value -> value != null && !value.isEmpty())
                 .collect(joining("\n"));
-        } else if (null != address) {
-            return Stream.of(
-                    address.getAddressLine1(),
-                    address.getAddressLine2(),
-                    address.getAddressLine3(),
-                    address.getPostTown(),
-                    address.getCounty(),
-                    address.getPostCode(),
-                    address.getCountry()
-                )
-                .filter(value -> value != null && !value.isEmpty())
-                .collect(joining("\n"));
+        } else if (!isConfidentialContactDetails() && null != address) {
+            return getApplicantAddress();
         }
-
         return null;
     }
 
     @JsonIgnore
-    public String getPostalAddress() {
+    public String getCorrespondenceAddressWithoutConfidentialCheck() {
         if (isRepresented()) {
-            return solicitor.getAddress();
+            return Stream.of(
+                    Optional.ofNullable(solicitor.getOrganisationPolicy())
+                        .map(OrganisationPolicy::getOrganisation).map(Organisation::getOrganisationName).orElse(null),
+                    solicitor.getAddress()
+                ).filter(value -> value != null && !value.isEmpty())
+                .collect(joining("\n"));
+        } else if (null != address) {
+            return getApplicantAddress();
```

Comment:
changing from using AddressUtil.getPostalAddress to getApplicantAddress means we don't have anything to cope with making the address fit in the envelope window moving forward

Reply:
We're barely using that method anyway now - the letterPrinter handles everything moved to a docPack, and it doesn't touch it.  Whenever we wrap a long line we push the postcode down.  This was partially compensated for by leaving out the country line for UK addresses, but we won't be doing that moving forward.  It runs the risk of pushing the postcode off the bottom of the window, and the postcode is the most important part of the address.